src/ResourceManager/RecoveryServices.Backup/Cmdlets/ProtectionPolicy/GetAzureRmRecoveryServicesPolicy.cs

```diff
@@ -49,15 +51,28 @@ public class GetAzureRmRecoveryServicesProtectionPolicy : RecoveryServicesBackup
 
         public override void ExecuteCmdlet()
         {
-            // TBD section below
-
             base.ExecuteCmdlet();
 
+            string rgName = "";  // TBD
+            string resourceName = "";  // TBD
+            List<AzureRmRecoveryServicesPolicyBase> respList = new List<AzureRmRecoveryServicesPolicyBase>();
```

Comment:
We should remove passing of ResourceName and ResourceGroupName from cmdlet as we are not making it optional param, hydra adapter should take these values from vault context.